generators/server/templates/src/main/java/package/config/_DatabaseConfiguration.java

```diff
@@ -208,22 +207,15 @@ public ValidatingCouchbaseEventListener validatingCouchbaseEventListener() {
     @Bean
     public LocalValidatorFactoryBean validator() {
         return new LocalValidatorFactoryBean();
-    }
+    }<% } if (databaseType === 'mongodb') { %>
 
-    @Bean<% if (databaseType === 'couchbase') { %>(name = BeanNames.COUCHBASE_CUSTOM_CONVERSIONS)<% } %>
-    public CustomConversions customConversions() {
-        List<Converter<?, ?>> converters = new ArrayList<>();<% if (databaseType === 'couchbase') { %>
-        converters.add(ZonedDateTimeToLongConverter.INSTANCE);
-        converters.add(NumberToLocalDateTimeConverter.INSTANCE);
-        converters.add(BigIntegerToStringConverter.INSTANCE);
-        converters.add(StringToBigIntegerConverter.INSTANCE);
-        converters.add(BigDecimalToStringConverter.INSTANCE);
-        converters.add(StringToBigDecimalConverter.INSTANCE);
-        converters.add(StringToByteConverter.INSTANCE);<% } else { %>
+    @Bean
+    public MongoCustomConversions customConversions() {
```

Comment:
`CustomConversions` is deprecated -> use the repo specific `MongoCustomConversions` and `CouchbaseCustomConversions`